aspnetcore/tutorials/razor-pages/page.md

```diff
@@ -59,9 +69,21 @@ The `DisplayNameFor` HTML Helper inspects the `Title` property referenced in the
 
 The `@model` directive specifies the type of the model passed to the Razor Page. In the preceding example, the `@model` line makes the `PageModel`-derived class available to the Razor Page. The model is used in the `@Html.DisplayNameFor` and `@Html.DisplayFor` [HTML Helpers](/aspnet/mvc/overview/older-versions-1/views/creating-custom-html-helpers-cs#understanding-html-helpers) on the page.
 
+### HTML Helpers
```

Comment:
```suggestion
```
I'd remove that line, it fits in with the preceding and you're not talking about HTML Helpers.